kadi_apps/blueprints/find_attitude/find_attitude.py

```diff
@@ -130,9 +130,13 @@ def index():
     # Update some constraints to their default values if not set.
     for key, value in DEFAULT_CONSTRAINTS.items():
         form_val = request.form.get(key)
-        if form_val is not None and form_val.strip() != "":
-            value = float(form_val)
-            context[key] = value
+        if form_val is not None and form_val.strip() == "":
```

Comment:
The `==` is a change in logic from the original and looks like it can't work. So I'm very confused that the live site appears to be working as expected.

Reply:
Yep, that's weird.  I can confirm that it looks like `/export/servers/kadi-test/ska3/lib/python3.10/site-packages/kadi_apps/blueprints/find_attitude/find_attitude.py` includes this newly added bug and still seems to basically work, so I should probably reexamine this whole chunk of logic if it wasn't really doing what I thought.